tests/testqualification/qualification/qual_workflows/1_mappings/ENROLL.yaml

```diff
@@ -0,0 +1,28 @@
+meta:
+  Type: Mapped
+  ID: ENROLL
+  Description: Enrollment Data Mapping 
+  Priority: 1
+spec: 
+  Raw_ENROLL:
+    studyid:
+      required: true
+      type: character
+    invid:
+      required: true
+      type: character
+    country:
+      required: true
+      type: character
+    subjectid:
+      required: yes
```

Comment:
@lauramaxwell Is "yes" an allowed value for required? This was directly taken from the `inst/workflows/1_mappings` ?

Reply:
technically, yes because `yaml::read_yaml` reads "yes" in as logical `TRUE`, and "no"` as a logical `FALSE`. I think this came about from the programmatic splitting up of these mapping files, and some of the defaults for `write_yaml()`. I might go in and change them for consistency, but it works either way